guides/chain-registration/sidechain/aggregatedBFTWeight.js

```diff
@@ -1,7 +1,7 @@
 const { apiClient } = require('@liskhq/lisk-client');
 
 (async () => {
-	// Update the path to point to your sidechain app data folder
+	// Update the path to point to your Lisk app's data folder
 	const sidechainClient = await apiClient.createIPCClient('~/.lisk/hello_client');
 	const sidechainNodeInfo = await sidechainClient.invoke('system_getNodeInfo');
```

Comment:
Please update the variable names, too, where needed.

```suggestion
	const blockchainClient = await apiClient.createIPCClient('~/.lisk/hello_client');
	const blockchainNodeInfo = await blockchainClient.invoke('system_getNodeInfo');
```

Reply:
The code changes suggested in this comment were incorporated into the original code, but I have now restored the code to its original state based on your latest suggestion.